src/components/custom-antd/FullWidthSpace/FullWidthSpace.styled.tsx

```diff
@@ -1,7 +1,13 @@
-import {Space as AntdSpace} from 'antd';
+import {Space as AntdSpace, SpaceProps} from 'antd';
 
 import styled from 'styled-components';
 
-export const StyledSpace = styled(AntdSpace)`
+export interface FullWidthSpaceProps extends SpaceProps {
+  justify?: 'flex-start' | 'flex-end' | 'center' | 'space-around' | 'space-between';
+}
+
+export const StyledSpace = styled(AntdSpace)<{justify: FullWidthSpaceProps['justify']}>`
```

Comment:
Probably `$justify`, to ensure it will not be passed to `AntdSpace`

Reply:
Agree